src/metadata/Metadata.ts

```diff
@@ -220,6 +230,18 @@ export class Metadata extends MarketplaceApi {
         return result
     }
 
+    public async retrieveDDOFromExternalBackend(immutableUrl: string): Promise<DDO> {
+        if (immutableUrl.startsWith('cid://'))  {
+            return await this.nevermined.utils.fetch
+                .fetchCID(immutableUrl)
+                .then((response: string) => {
+                    return DDO.deserialize(response)
+                })
+        } else if (immutableUrl.startsWith('http')) {
```

Comment:
a url that starts with `http` is not really immutable

Reply:
there could be used a url to a public gateway that exposes an immutable content. For example: 
https://gateway.ipfs.io/ipfs/bafybeifx7yeb55armcsxwwitkymga5xf53dxiarykms3ygqic223w5sk3m

Reply:
I know but the only immutable part there is the cid, the gateway may change

Reply:
okay, I will remove it by now